packages/@aws-cdk/aws-autoscaling/test/test.auto-scaling-group.ts

```diff
@@ -1088,24 +1088,136 @@ export = {
     test.done();
   },
 
-  'throw if notification and notificationsTopics are both configured'(test: Test) {
+  'test emitAllGroupMetrics adds a single MetricsCollection with no Metrics specified'(test: Test) {
     // GIVEN
     const stack = new cdk.Stack();
     const vpc = mockVpc(stack);
-    const topic = new sns.Topic(stack, 'MyTopic');
+    const asg = new autoscaling.AutoScalingGroup(stack, 'MyStack', {
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.M4, ec2.InstanceSize.MICRO),
+      machineImage: new ec2.AmazonLinuxImage(),
+      vpc,
+    });
 
-    // THEN
-    test.throws(() => {
-      new autoscaling.AutoScalingGroup(stack, 'MyASG', {
-        instanceType: ec2.InstanceType.of(ec2.InstanceClass.M4, ec2.InstanceSize.MICRO),
-        machineImage: new ec2.AmazonLinuxImage(),
-        vpc,
-        notificationsTopic: topic,
-        notifications: [{
-          topic,
-        }],
-      });
-    }, 'Can not set notificationsTopic and notifications, notificationsTopic is deprected use notifications instead');
+    // When
+    asg.emitAllGroupMetrics();
+
+    // Then
+    expect(stack).to(haveResource('AWS::AutoScaling::AutoScalingGroup', {
+      MetricsCollection: [
+        {
+          'Granularity': '1Minute',
+          Metrics: ABSENT,
+        },
+      ],
+    }));
+    test.done();
+  },
+
```

Comment:
Need to add test cases when specific metrics are specified in the constructor but `emitAllGroupMetrics()` is called and vice versa.